tools/prefetch_test.py

```diff
@@ -13,7 +13,7 @@ def prefetch_test(deno_exe):
     deno_dir = mkdtemp()
     try:
         t = os.path.join(tests_path, "006_url_imports.ts")
-        output = run_output([deno_exe, "prefetch", t],
+        output = run_output([deno_exe, "fetch", t],
```

Comment:
Could you also rename the file `tools/prefetch_test.py` to `tools/fetch_test.py`?

Reply:
Sure, no problem.